Example/Example/MoreAttachmentRender.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  MoreAttachmentRender.swift
+//  AztecExample
+//
+//  Created by Jorge Leandro Perez on 3/22/17.
+//  Copyright © 2017 Automattic Inc. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Aztec
+
+
+// MARK: - MoreAttachmentRender: Renders More Comments!
+//
+class MoreAttachmentRender {
```

Comment:
Couldn't this class be part of the open API library to allows users of the library to use it, and avoid code replication.